### DISCUSSION TITLE
Application showing completely black screen after initial config and test print

### DISCUSSION THREAD

**Discussion by iacsha** [porrey/Virtual-ZPL-Printer#20] [Q&A]
Hi Group,

I am using VMware Workstation v16 with Windows 10 Virtual Machine
I installed the Virtual ZPL Printer application and did initial setup to create a profile.
Once I clicked 'Test Print' the application screen is now completely black.

I have restarted the Virtual Machine and uninstalled and reinstalled the application.

Has anyone experienced this happening?
![VirtualZPL-black](https://user-images.githubusercontent.com/122627209/212360305-8f38816a-a4ca-4f7e-88f5-e3a8ba3eb2ec.PNG)

**Comment by FranBellCast** [comment]
Same issue. Nothing is working.

**Comment by iacsha** [comment]
Hi, unfortunately still having this issue. Attached is the log file. Any ideas??
[VirtualZPLLog.txt](https://github.com/user-attachments/files/17476522/VirtualZPLLog.txt)

**Comment by porrey** [comment]
I haven't seen this issue. I don't see anything abnormal in the logs. 

Does the screen show when the application starts and then goes black after receiving the first label?

**Reply by iacsha** [reply]
I was able to get it working by
Uninstall, I used Revo Uninstaller to get the remaining files and Registry Keys
Delete the Log and Database folder, needed to do manual
Reinstall

Now working!
